### PR TITLE
[88414230] Added random daily meetings between the offices.

### PR DESCRIPTION
**Pull request by keiter**

### CODE REVIEW

File: src/awesome_o/slack.clj [hunk 1]
```diff
@@ -53,10 +62,5 @@
...
     (doseq [person (state/persons-born-today)]
       (say (format "Today is @%s's birthday! Happy birthday!" person)))
-    (when (time/monday-today?)
-      (let [devs (shuffle (state/available-devs))
-            gbgs (shuffle (state/get-available-people-in-location "göteborg"))
-            meeting-master (pingify [(rand-nth gbgs)])
-            honeybadgers (->> devs (take 2) pingify)]
-        (say (str "Honeydager monday! ping: " honeybadgers) :channel "dev")
-        (say (str "Todays meeting master for dev this week is " meeting-master))))))
+    (when (time/monday-today?) monday-announcements)
```
Comment (Jell) [inline src/awesome_o/slack.clj]: OK I guess almost caricatural first PR review comment: you forgot a pair of parenthesis around "monday-announcements"

Comment (Jell) [inline src/awesome_o/slack.clj]: we might want to test that someday

Comment (keiter) [inline src/awesome_o/slack.clj]: bad haskell habit, assumed that zero-arity function is already applied :P

Comment (keiter) [inline src/awesome_o/slack.clj]: but yeah, test coverage could be better

Comment (Jell) [inline src/awesome_o/slack.clj]: sorry, no curry for you!

**Pull request merged**